runtime/cni_integ_test.go

```diff
@@ -39,8 +39,7 @@ import (
 )
 
 func TestCNISupport_Isolated(t *testing.T) {
-	prepareIntegTest(t)
-
+	prepareIntegTest(t, withJailer())
```

Comment:
I just looked in buildkite, it seems like this isn't working as expected, `TestCNISupport_Isolated` is still running w/ the noop jailer according to the logs, do we need to update the `CreateVM` request in here to specify a non-nil jailer?

Reply:
Doh, fixed.